puppeteer-tests/src/performance-test.ts

```diff
@@ -473,20 +472,27 @@ const getFrameData = (
     } else {
       const frameDelta = (endEvent.ts - startEvent.ts) / 1000
       frameTimes[frameCounter] = frameDelta
-      totalFrameTimes += frameDelta
     }
   }
 
   let frameTimesFixed = frameTimes.map((x) => Number(x.toFixed(1)))
   const sortedFrameTimes = frameTimesFixed.sort((a, b) => a - b)
+  // Trims the top 5% off.
+  let withoutOutliersLength = Math.round(sortedFrameTimes.length * 0.95)
+  // Ensure we always have at least one result.
+  if (withoutOutliersLength <= 0) {
+    withoutOutliersLength = 1
+  }
+  const frameTimesWithoutOutliers = sortedFrameTimes.slice(0, withoutOutliersLength)
+  let totalFrameTimes = 0
```

Comment:
This is definitely not going to produce the right `frameAvg`! But that doesn't matter because we don't even use `frameAvg`, so in the bin with it!

Reply:
Ah, I've fixed this in 31ff7dbe3, we should probably remove that in a different PR.